app/e2e/tests/dashboard.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { test, expect } from '@playwright/test';
+
+test('Dashboard has button to create event.', async ({ page }) => {
```

Comment:
I suggest wrapping tests in describe blocks like the landing tests are. Also might be a good idea to break up dashboard tests into multiple files under a dashboard folder like the landing tests. Playwright can only execute tests in parallel when they are in separate files.